test/ping-pong/ping-pong.test.mjs

```diff
@@ -1,156 +1,324 @@
 import config from 'config';
-import compose from 'docker-compose';
-import yaml from 'js-yaml';
-import fs from 'fs';
-import { Docker } from 'docker-cli-js';
-import Web3 from 'web3';
 import { expect } from 'chai';
+import { retrieveL2Balance } from '../utils.mjs';
 import Nf3 from '../../cli/lib/nf3.mjs';
-import { userTest, proposerTest, setParametersConfig } from './index.mjs';
+import {
+  proposerTest,
+  setParametersConfig,
+  getStakeAccount,
+  getCurrentProposer,
+  simpleUserTest,
+} from './index.mjs';
 
-const { mnemonics, signingKeys } = config.TEST_OPTIONS;
+const { mnemonics, signingKeys, clientApiUrls, optimistApiUrls, optimistWsUrls, fee } =
+  config.TEST_OPTIONS;
 const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
+const { TEST_ERC20_ADDRESS } = process.env;
 
-let result;
+const nf3Users = [];
+let blockStake;
+const listAddresses = [];
+const listTransfersTotal = [];
+const TEST_LENGTH = 4;
+const value = 10;
```

Comment:
Should this be a random sequence, rather than a single value, to avoid the issue of a false pass if both users fail to send?

Reply:
This is only used for the initial deposits to be included in a block. The first part of the test creates TEST_LENGTH deposits of 10 in this case. Then for every loop of the TEST_LENGTH we do 1 transfer and 1 deposit of a random value between 1 and 9 in this case. 
```
// Returns a random integer from 1 to value - fee
const valueToTransfer = Math.floor(Math.random() * (value - fee)) + 1; 
```
I do like this because I defined 1 for the fee. So then with the first deposits included in a block, every user will have enough to cover the randoms values + 1 of the fee and at maximum it will be 10 every loop.